CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # React Native Module Changelog
 
+## Version 13.1.2 - February 9, 2022
+Minor release that updates the Airship SDKs and fixes several issues.
+
+### Changes
+- Updated iOS SDK to 16.2.0
+- Updated Android SDK to 16.2.0
+- Added a method `getSubscriptionLists` to get the subscription lists
+- Added a method `setNotificationOptions` to set the notification options for iOS
+- Fixed iOS message center display when `autoLaunchMessageCenter` is set to false
+- Fixed the preference center config to return the items under the key `items`
```

Comment:
Added a method `getNotificationStatus` to get more context on the current state of notifications